app/src/test/java/com/appunite/loudius/ui/reviewers/ReviewersViewModelTest.kt

```diff
@@ -91,104 +98,160 @@ class ReviewersViewModelTest {
         fun `GIVEN correct initial values WHEN init THEN pull request number is correct`() {
             viewModel = createViewModel()
 
-            assertEquals("correctPullRequestNumber", viewModel.state.pullRequestNumber)
+            expectThat(viewModel.state)
+                .get(ReviewersState::pullRequestNumber)
+                .isEqualTo("correctPullRequestNumber")
```

Comment:
**Suggestion** That could be onelined

Reply:
I think making this in separate lines makes easier to understand what is happening line by line